rotkehlchen/accounting/history_base_entries.py

```diff
@@ -196,33 +214,31 @@ def _process_swap(
                 fee_taxable = True
                 fee_taxable_amount_ratio = trade_taxable_amount / out_event.balance.amount
 
-            self.pot.add_out_event(
-                event_type=AccountingEventType.FEE,
-                notes=fee_event.notes,  # type: ignore [arg-type]  # notes exist here
-                location=fee_event.location,
-                timestamp=timestamp,
-                asset=fee_event.asset,
-                amount=fee_event.balance.amount,
-                taxable=fee_taxable,
-                given_price=fee_price,
-                # By setting the taxable amount ratio we determine how much of the fee
-                # spending should be a taxable spend and how much free.
-                taxable_amount_ratio=fee_taxable_amount_ratio,
-                count_cost_basis_pnl=True,
-                count_entire_amount_spend=True,
-                extra_data=extra_data,
-            )
-            consumed_events = 3
+            def add_fee_event() -> None:
```

Comment:
same here. And this is exactly why you have to add so many type ignores here ... this is a hack. Undo